source/developers/file-transfer-protocol.rst

```diff
@@ -14,219 +14,11 @@ File Transfer Protocol (FTP/SFTP)
    :local:
    :depth: 1
 
-.. versionadded:: MinIO RELEASE.2023-04-20T17-56-55Z
+.. cond:: not k8s
 
-Overview
---------
+   .. include:: /includes/linux/file-transfer-protocol-not-k8s.rst
 
-You can use the File Transfer Protocol (``FTP``) to interact with the objects on a MinIO deployment.
+.. cond:: k8s and not (openshift or eks or gke or aks)
```

Comment:
I think this is the correct `cond::` for `k8s` and `everything else`. The other targets directly link to the Linux version of this page rather than having their own.